.eslintrc.json

```diff
@@ -50,6 +50,7 @@
     "import/no-extraneous-dependencies": ["error", {
       "devDependencies": true
     }],
-    "import/no-cycle": "off"
+    "import/no-cycle": "off",
+    "linebreak-style": "off"
```

Comment:
Since I'm developing on Windows, if I don't enable this option, I get an error on every line ) However, any windows git client converts crlf to lf during any git commit

Reply:
As I said, you should fix it in the IDE settings.

Reply:
а кто нибуть на практике пробовал так работать.  потому что для windows crlf возникает на всех шагах - когда файлы из git приходят, когда редактируешь любым не специально настроенным редактором, генераторы кода тоже с таким переносом пишут.  так что просто настройки IDE задачу увы не решают